Header header
UserRGBD[] users
uint32 image_w
uint32 image_h
sensor_msgs/PointCloud2 cloud
uint8 compute_depth
